client-spark/common/src/main/java/org/apache/spark/shuffle/reader/RssShuffleDataIterator.java

```diff
@@ -38,8 +38,12 @@
 
 import org.apache.uniffle.client.api.ShuffleReadClient;
 import org.apache.uniffle.client.response.CompressedShuffleBlock;
-import org.apache.uniffle.common.RssShuffleUtils;
-import org.apache.uniffle.common.exception.RssException;
+import org.apache.uniffle.client.util.RssClientConfig;
+import org.apache.uniffle.common.compression.CompressionFactory;
+import org.apache.uniffle.common.compression.Decompressor;
+import org.apache.uniffle.common.config.RssConf;
+
+import static org.apache.spark.shuffle.RssSparkConfig.RSS_WRITER_BUFFER_SIZE;
```

Comment:
Should we avoid static import here？

Reply:
Yes. We'd better to avoid this. And in later refactor, all client side's config should be involved in shareable `RssClientConf` class. This problem will not occur, which is mentioned in #200

Reply:
Could we make it consistent with other factory methods in our project?

Reply:
I think I missunderstand you meaning. So we just to use `RssSparkConfig.RSS_WRITER_BUFFER_SIZE.defaultValueString()` to avoid static import, right?

Reply:
Done

Reply:
> I think I missunderstand you meaning. So we just use `RssSparkConfig.RSS_WRITER_BUFFER_SIZE.defaultValueString()` to avoid static import, right?

Yes